macosx.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+# This script contains common code for building PyNE on various Debian-derived systems
+#
+
+function check_repo() {
+
+    repo_name=$1
+
+    if [ -d $repo_name ] ; then
```

Comment:
Why no curly braces as there are in the ubuntu script?  They may not be strictly necessary but good practice. (This comment applies in many places throughout.)